lib/face_control/checkers/comments.rb

```diff
@@ -0,0 +1,32 @@
+require 'face_control/comment'
+
+module FaceControl
+  module Checkers
+    class Comments
+      def relevant_globs
+        %w(*.rb)
+      end
```

Comment:
I think that `TODO`s and `FIXME`s are harmful in any file types.

I have just made a commit to the `develop` branch that makes this method optional. You can remove it now.